package.json

```diff
@@ -38,7 +38,8 @@
     "typescript": "^3.9.7"
   },
   "overrides": {
-    "@types/prettier": "2.6.0"
+    "@types/prettier": "2.6.0",
+    "@types/express-serve-static-core": "ts3.9"
```

Comment:
Are these types needed here? It does not seem like this specific package.json has a dependency on Express. Perhaps this is just a part of how Lerna works that I'm not familiar with?

Reply:
I don't think it's strictly necessary here. Npm overrides gives me a headache at times and wanted to be extra sure that the wrong version can't be anywhere in the path especially since we run TypeScript typically from the root. There's definitely some overall dependency cleanup we can do.